test/unit/controls.test.js

```diff
@@ -1,114 +1,122 @@
     on: noop,
     ready: noop,
     tech_: {
-      'featuresVolumeControl': false
+      featuresVolumeControl: false
     },
-    volume: function(){},
-    muted: function(){},
-    reportUserActivity: function(){}
+    volume() {},
+    muted() {},
+    reportUserActivity() {}
   };
 
-  volumeControl = new VolumeControl(player);
-  muteToggle = new MuteToggle(player);
+  const volumeControl = new VolumeControl(player);
+  const muteToggle = new MuteToggle(player);
 
-  ok(volumeControl.el().className.indexOf('vjs-hidden') >= 0, 'volumeControl is not hidden');
-  ok(muteToggle.el().className.indexOf('vjs-hidden') >= 0, 'muteToggle is not hidden');
+  QUnit.ok(volumeControl.el().className.indexOf('vjs-hidden') >= 0,
+  'volumeControl is not hidden');
+  QUnit.ok(muteToggle.el().className.indexOf('vjs-hidden') >= 0,
+  'muteToggle is not hidden');
 });
 
-test('should test and toggle volume control on `loadstart`', function(){
-  var noop, listeners, player, volumeControl, muteToggle, i;
-  noop = function(){};
-  listeners = [];
-  player = {
+QUnit.test('should test and toggle volume control on `loadstart`', function() {
+  let i;
+
+  const noop = function() {};
+  const listeners = [];
+  const player = {
     id: noop,
-    on: function(event, callback){
+    on(event, callback) {
       // don't fire dispose listeners
       if (event !== 'dispose') {
         listeners.push(callback);
       }
     },
     ready: noop,
-    volume: function(){
+    volume() {
       return 1;
     },
-    muted: function(){
+    muted() {
       return false;
     },
     tech_: {
-      'featuresVolumeControl': true
+      featuresVolumeControl: true
     },
-    reportUserActivity: function(){}
+    reportUserActivity() {}
   };
 
-  volumeControl = new VolumeControl(player);
-  muteToggle = new MuteToggle(player);
+  const volumeControl = new VolumeControl(player);
+  const muteToggle = new MuteToggle(player);
 
-  equal(volumeControl.hasClass('vjs-hidden'), false, 'volumeControl is hidden initially');
-  equal(muteToggle.hasClass('vjs-hidden'), false, 'muteToggle is hidden initially');
+  QUnit.equal(volumeControl.hasClass('vjs-hidden'), false,
+  'volumeControl is hidden initially');
+  QUnit.equal(muteToggle.hasClass('vjs-hidden'), false, 'muteToggle is hidden initially');
 
-  player.tech_['featuresVolumeControl'] = false;
+  player.tech_.featuresVolumeControl = false;
   for (i = 0; i < listeners.length; i++) {
     listeners[i]();
   }
 
-  equal(volumeControl.hasClass('vjs-hidden'), true, 'volumeControl does not hide itself');
-  equal(muteToggle.hasClass('vjs-hidden'), true, 'muteToggle does not hide itself');
+  QUnit.equal(volumeControl.hasClass('vjs-hidden'), true,
+  'volumeControl does not hide itself');
+  QUnit.equal(muteToggle.hasClass('vjs-hidden'), true, 'muteToggle does not hide itself');
 
-  player.tech_['featuresVolumeControl'] = true;
+  player.tech_.featuresVolumeControl = true;
   for (i = 0; i < listeners.length; i++) {
```

Comment:
take the `let i` from the top of this file and instead put it in this for statement `for (let i = 0;...`